docker-compose.yml

```diff
@@ -1,4 +1,40 @@
 version: "3.3"
+x-environment: &environment
+  environment:
+    # IF YOU ARE USING DOCKER, PLEASE MAKE SURE YOU DO NOT HAVE A .env FILE IN THE ROOT DIRECTORY.
+    # IF you do, please rename it to anything else. This is because docker-compose will load the .env file
+    # and throw an error as it is according to the NextJs 13.4.19 versions and docker will not recognise it.
+
+    # This should be the same as below if you are running via docker compose up
+    WEBAPP_URL: http://formbricks:3000
+
+    # PostgreSQL DB for Formbricks to connect to
+    DATABASE_URL: "postgresql://postgres:postgres@postgres:5432/formbricks?schema=public"
+
+    # Uncomment to enable a dedicated connection pool for Prisma using Prisma Data Proxy
+    # Cold boots will be faster and you'll be able to scale your DB independently of your app.
+    # @see https://www.prisma.io/docs/data-platform/data-proxy/use-data-proxy
+    # PRISMA_GENERATE_DATAPROXY=true
+    PRISMA_GENERATE_DATAPROXY:
+
+    # NextJS Auth
+    # @see: https://next-auth.js.org/configuration/options#nextauth_secret
+    # You can use: `openssl rand -base64 32` to generate one
+    NEXTAUTH_SECRET: MoPAA5lQLzx9J8xRa8JWpIwrB4U7STgVImLzI4Xtjag=
```

Comment:
As we are now revamping the approach, should we maybe add a mandatory command to generate a secret first? (like we have in the setup script)

Reply:
In our docs right?

Reply:
Yes, in the docs :-) 
🙏

Reply:
Added in docs